src/cache/LocalStorageCache.spec.ts

```diff
@@ -0,0 +1,51 @@
+import {LocalStorageCache} from "./LocalStorageCache";
+
+describe("LocalStorageCache", () => {
+    it("is defined", () => {
+        expect(LocalStorageCache).toBeDefined();
+    });
+
+    const store = new LocalStorageCache();
+
+    describe("setItem()", () => {
+        it("is function", () => {
+            expect(typeof store.setItem).toBe("function");
+        });
+
+        it("accepts key, value and ttl; returns nothing", () => {
+            expect(store.setItem("key", "value", 5)).toBeUndefined();
+        });
+    });
+
+    describe("getItem()", () => {
+        it("is function", () => {
+            expect(typeof store.getItem).toBe("function");
+        });
+
+        it("returns the saved value", () => {
+            expect(store.getItem("key")).toBe("value");
```

Comment:
Every test should be runnable by itself or in any combination with other tests, possibly running multiple times and in any order. Always having the exact same result. This test does not. :)